src/main/java/ch/njol/skript/entity/BoatChestData.java

```diff
@@ -24,28 +24,43 @@
 import ch.njol.skript.lang.Literal;
 import ch.njol.skript.lang.SkriptParser;
 import org.bukkit.Material;
-import org.bukkit.TreeSpecies;
+import org.bukkit.entity.Boat;
 import org.bukkit.entity.ChestBoat;
 import org.bukkit.inventory.ItemStack;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.Locale;
 import java.util.Random;
 
 public class BoatChestData extends EntityData<ChestBoat> {
 
+	private static final Boat.Type[] types = Boat.Type.values();
```

Comment:
Boat.Type was added in Bukkit 1.19, how is this going to handle previous versions?

Reply:
Well, Skript 2.10 is only going to support the 3 latest versions right? So 1.21, 1.20 and 1.19 no?

Reply:
Ahh, I wasn't sure about that. good to know! Carry on!